app/src/main/java/me/ash/reader/domain/service/GoogleReaderRssService.kt

```diff
@@ -228,7 +228,7 @@ class GoogleReaderRssService @Inject constructor(
 
                 // 2. Fetch folder and subscription list
                 googleReaderAPI.getSubscriptionList()
-                    .subscriptions.groupBy { it.categories?.first() }
+                    .subscriptions.sortedBy { it.categories?.sortedBy { it.label }?.first()?.label }.groupBy { it.categories?.first() }
```

Comment:
This seems a bit complicated for me. Though I don't use Google Reader, I guess we only need the first (and the only) element in `categories` here. And we can sort the map after we group the list by the category.